eventlet/wsgi.py

```diff
@@ -799,6 +805,7 @@ def __init__(self,
         self.url_length_limit = url_length_limit
         self.debug = debug
         self.socket_timeout = socket_timeout
+        self.idle_timeout = idle_timeout
```

Comment:
```suggestion
        self.idle_timeout = idle_timeout or socket_timeout
```

doc says default is to use socket_timeout

Reply:
Sure; then we can also get rid of the `if self.idle_timeout:` guards -- I mainly went this way because I wasn't sure whether there'd be some penalty for calling `settimeout` a bunch, or if so, how big it was.

Reply:
Yes, syscall penalty is non-negligible (around 2us best case). Also I think idle timeout is on verge of DoS resilience, and the only faster way I know is `monotonic() > deadline` which gets time via VDSO without syscall in best case.

I think we should document the cost and recommend (as always) external reverse proxy, which downgrades disabled keepalive from DoS to internal config/resource management.